timer-blink/timed-blink.c

```diff
@@ -0,0 +1,92 @@
+#include <avr/io.h>
+#include <util/delay.h>
+#include <stdbool.h>
+
+bool light_on;
+
+// Shift the passed bit on the port data direction register to set it as output
+void setup_bit_as_output(int bit)
+{
+    if (bit == 5)
+    {
+        DDRB |= (1 << DDB5);
+    }
+    else if (bit == 4)
+    {
+        DDRB |= (1 << DDB4);
+    }
+    else if (bit == 2)
+    {
+        DDRB |= (1 << DDB4);
```

Comment:
Wrong bit

Reply:
Good catch--killed the whole branching logic anyway.